manifests/backend/resources.pp

```diff
@@ -1,152 +1,101 @@
-    old_password => $::sensu::backend::agent_old_password,
-    groups       => ['system:agents'],
-  }
-
-  sensu_cluster_role { 'admin':
-    ensure => 'present',
-    rules  => [
-      {
-        'verbs'     => ['*'],
-        'resources' => [
-          'assets',
-          'checks',
-          'entities',
-          'extensions',
-          'events',
-          'filters',
-          'handlers',
-          'hooks',
-          'mutators',
-          'silenced',
-          'roles',
-          'rolebindings',
-        ],
-      },
-      {
-        'verbs'     => ['get', 'list'],
-        'resources' => ['namespaces'],
-      },
-    ],
-  }
-  sensu_cluster_role { 'cluster-admin':
-    ensure => 'present',
-    rules  => [
-      {
-        'verbs'     => ['*'],
-        'resources' => ['*'],
-      },
-    ],
-  }
-  sensu_cluster_role { 'edit':
-    ensure => 'present',
-    rules  => [
-      {
-        'verbs'     => ['*'],
-        'resources' => [
-          'assets',
-          'checks',
-          'entities',
-          'extensions',
-          'events',
-          'filters',
-          'handlers',
-          'hooks',
-          'mutators',
-          'silenced',
-        ],
-      },
-      {
-        'verbs'     => ['get', 'list'],
-        'resources' => ['namespaces'],
-      },
-    ],
-  }
-  sensu_cluster_role { 'system:agent':
-    ensure => 'present',
-    rules  => [
-      {
-        'verbs'     => ['*'],
-        'resources' => ['events'],
-      },
-    ],
-  }
-  sensu_cluster_role { 'system:user':
-    ensure => 'present',
-    rules  => [
-      {
-        'verbs'     => ['get', 'update'],
-        'resources' => ['localselfuser'],
-      },
-      {
-        'verbs'     => ['get', 'list'],
-        'resources' => ['namespaces'],
-      },
-    ],
-  }
-  sensu_cluster_role { 'view':
-    ensure => 'present',
-    rules  => [
-      {
-        'verbs'     => ['get', 'list'],
-        'resources' => [
-          'assets',
-          'checks',
-          'entities',
-          'extensions',
-          'events',
-          'filters',
-          'handlers',
-          'hooks',
-          'mutators',
-          'silenced',
-          'namespaces',
-        ],
-      },
-    ],
-  }
-
-  sensu_cluster_role_binding { 'cluster-admin':
-    ensure   => 'present',
-    role_ref => 'cluster-admin',
-    subjects => [
-      {
-        'type' => 'Group',
-        'name' => 'cluster-admins'
-      },
-    ],
-  }
-  sensu_cluster_role_binding { 'system:agent':
-    ensure   => 'present',
-    role_ref => 'system:agent',
-    subjects => [
-      {
-        'type' => 'Group',
-        'name' => 'system:agents'
-      },
-    ],
-  }
-  sensu_cluster_role_binding { 'system:user':
-    ensure   => 'present',
-    role_ref => 'system:user',
-    subjects => [
-      {
-        'type' => 'Group',
-        'name' => 'system:users'
-      },
-    ],
+  $::sensu::backend::ad_auths.each |$name, $ad_auth| {
```

Comment:
These should be in alphabetical order

Reply:
Are they not?  I did "ls -la" on lib/puppet/types and did it in that order.